app/jobs/views.py

```diff
@@ -0,0 +1,33 @@
+# Third-Party imports
+from django.contrib import messages
+from django.shortcuts import render
+from django.template.response import TemplateResponse
+
+# gitcoin-web imports
+from .forms import JobForm
+
+
+def list_jobs(request):
+    context = dict()
+    return TemplateResponse(request, 'jobs/list.html', context=context)
+
+
+def job_detail(request, pk):
+    context = {
+        'pk': pk
```

Comment:
🤔

Reply:
Should we handle any kind of update functionality against details and use the form you've already created?